code/modules/clothing/spacesuits/rig/suits/light.dm

```diff
@@ -62,10 +62,12 @@
 
 /obj/item/clothing/suit/lightrig/hacker
 	siemens_coefficient = 0.4
+	var/holder = null //holder is required for suits
 
 /obj/item/clothing/shoes/lightrig/hacker
 	siemens_coefficient = 0.4
 	flags = NOSLIP //All the other rigs have magboots anyways, hopefully gives the hacker suit something more going for it.
+	var/magboot_state = null //magboot_state must be defined for all shoes
```

Comment:
I think that will block a runtime at creation, as the rig backpack item expects magboots and a rig-spacesuit as boots and chestpiece respectivly, and assigns ``magboot_state`` for the magboot toggling and ``holder`` for the spacesuit breaching. But the addition of the vars should either be made to the lightrig type, not the hacker subtype, or a sanity check should be made before assinging them from the rig creation. The issue might also exist for rig unsealing, as it will check the expected magboots if they are on, but if byond gives you ``null`` when you try to access some non-existing var from an object, that should work.